superset/assets/spec/helpers/shim.js

```diff
@@ -46,9 +45,6 @@ global.XMLHttpRequest = global.window.XMLHttpRequest;
 
 global.sinon = require('sinon');
```

Comment:
In a future PR I think we can actually also remove sinon because `jest` has support for spies, stubs, etc. I doubt that's code-mod-able, tho.

Reply:
Totally agree. I didn't do it in the same PR to limit the scope. :) Yeah, for `sinon`, we need to modify things manually.